app/src/test/java/com/example/javadevelopersnairobi/javadevelopersnairobi/GitHubUsersTest.java

```diff
@@ -0,0 +1,17 @@
+package com.example.javadevelopersnairobi.javadevelopersnairobi;
+
+public class GitHubUsersTest {
+//	@Test
+//	public void TestGithubUsers(){
+//		String username = "wycliffkas";
+//		String github_link = "https://github.com/wycliffkas";
+//		String image_url = "https://avatars10.githubusercontent.com/u/25085146?v=4";
+//
+//		GithubUsers userDetails = new GithubUsers(username, github_link, image_url);
+//
+//		assertEquals(username,userDetails.getUsername());
+//		assertEquals(github_link,userDetails.getHtmlUrl());
+//		assertEquals(image_url,userDetails.getAvatarUrl());
+//
+//	}
+}
```

Comment:
@wycliffkas , hey this is not cool at all, you leave uncommented code like this 😫

Reply:
sorry about that, let me remove them